lib/pillar/type_convert/to_clickhouse_json.ex

```diff
@@ -51,6 +51,10 @@ defmodule Pillar.TypeConvert.ToClickhouseJson do
   def convert(param) when is_binary(param) do
     param
   end
+  
+  def convert({:json, param}) do
```

Comment:
@Voronchuk Hi!

Thanks for contribution.


Provide please some tests for this data type, as this one, for example: https://github.com/balance-platform/pillar/blob/master/test/pillar_test.exs#L296

Reply:
@sofakingworld I have added the requested features, with some extra tweaks for better support of JSON type (e.g. automatic Tuple conversion)